packages/coding-standards/Jetpack/ruleset.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Jetpack" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/squizlabs/PHP_CodeSniffer/master/phpcs.xsd">
+
+	<rule ref="PHPCompatibilityWP"/>
```

Comment:
Works best if there is a `<config name="testVersion" value="5.6-"/>` value set.

Reply:
My initial thought was leaving that up to implementing projects to implement, but upon reflecting on your review, let's be opinionated.

Moved config settings to the ruleset, removed them from the sample and Jetpack-Plugin in e64898095.